vizro-core/src/vizro/static/css/images.css

```diff
@@ -1,21 +0,0 @@
-img[src*="floating-right"] {
-  float: right;
-}
-
-img[src*="floating-left"] {
-  float: left;
-}
-
-img[src*="floating-center"] {
-  display: block;
-  float: none;
-  height: auto;
-  margin: auto;
-  max-width: 100%;
-}
```

Comment:
I decided to remove all of these as I think we don't need to introduce another shortcut layer. Bootstrap does have shortcut classNames for these already, which I would suggest using if you want to float images: https://getbootstrap.com/docs/5.0/content/images/

In the particular example in the docs, we unfortunately can't use them as we can't add classNames to the image markdown, see: https://stackoverflow.com/questions/22627458/how-to-add-class-in-image-markdown-in-ghost

In this case, I found it sufficient to add an example to the docs.

Reply:
Maybe in future we would extend markdown to allow for bootstrap class names but for now it's fine by me for users to do this manually like you suggest.